chords_mqtt/orchestrator.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+import paho.mqtt.client as mqtt
+import yaml
+import requests
+from .config import config as cfg
+# todo : add logging
+
+
+# The callback for when the client receives a CONNACK response from the server.
+def on_connect(client, userdata, flags, rc):
```

Comment:
Do we need the `userdata` and `flags` in the function signature? They are not being used in the function.